src/test/groovy/org/gradle/profiler/mutations/ApplyProjectDependencyChangeMutatorIntegrationTest.groovy

```diff
@@ -0,0 +1,101 @@
+package org.gradle.profiler.mutations
+
+import org.gradle.profiler.AbstractProfilerIntegrationTest
+import org.gradle.profiler.Main
+
+class ApplyProjectDependencyChangeMutatorIntegrationTest extends AbstractProfilerIntegrationTest {
+
+    def "successfully benchmarks gradle projects with dependency mutations with default values"() {
+        given:
+        instrumentedBuildScript()
+        println buildFile
+        def scenarioFile = file("performance.scenarios") << """
+            s1 {
+                tasks = assemble
+                apply-project-dependency-change-to {
+                    files = ["build.gradle"]
+                }
+            }
+        """
+
+        when:
+        new Main().run("--project-dir", projectDir.absolutePath, "--output-dir", outputDir.absolutePath, "--gradle-version", latestSupportedGradleVersion, "--scenario-file", scenarioFile.absolutePath, "--benchmark", "s1")
```

Comment:
Can we extract this into some sort of utility method?

Reply:
moved to runBenchmark method